script/functions.sh

```diff
@@ -11,6 +11,14 @@ export JRUBY_OPTS=${JRUBY_OPTS:-"--server -Xcompile.invokedynamic=false"}
 SPECS_HAVE_RUN_FILE=specs.out
 MAINTENANCE_BRANCH=`cat maintenance-branch`
 
+# Don't allow rubygems to pollute what's loaded. Also, things boot faster
+# without the extra load time of rubygems. Only works on MRI Ruby 1.9+
+if is_mri_192_plus; then
+  export RUBYOPT="--disable=gem"
+else
+  export RUBYOPT=""
```

Comment:
Doesn't look like it. I ran a quick test on Travis for my open rspec-support PR. I dumped `env` from the travis config via `bundle_install` and in `script/run_build` using `bundle exec ruby -rpp -e 'pp ENV'`. I don't see `RUBYOPT` set anywhere by CI.

You can see the output (I only ran Ruby 1.8.7, 2.2, and JRuby): https://travis-ci.org/rspec/rspec-support/builds/75377043